frontend/src/modules/layout/pinia/state.ts

```
@@ -0,0 +1,9 @@
+export interface FooterState {
+  visible: Boolean;
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)